action.yml

```diff
@@ -2,13 +2,7 @@
 
 name: Run MATLAB Command
 description: >-
-  Execute a MATLAB script, function, or statement. 
-  MATLAB exits with exit code 0 if the specified script, function, or statement
-  executes successfully without error. Otherwise, MATLAB terminates with a
-  nonzero exit code, which causes the build to fail. You can use the assert or
-  error functions in the command to ensure that builds fail when necessary. When
-  you use this task, all of the required files must be on the MATLAB search
-  path.
+  Execute a MATLAB script, function, or statement
```

Comment:
Run MATLAB scripts, functions, and statements

Reply:
Houman, you're the maestro